src/main/java/http/configure/Configuration.java

```diff
@@ -0,0 +1,24 @@
+package http.configure;
```

Comment:
Great job 👏  This is super clean code!

Reply:
It is missing unit tests though.

Reply:
Agreed - this is super easy to read and understand 👏